modules/vba_documents/lib/vba_documents/object_store.rb

```diff
@@ -30,6 +30,10 @@ def download(object_version, path)
                         response_target: path)
     end
 
+    def delete(object_key)
```

Comment:
Whoops, looks like this will not actually delete versioned objects :( 
https://docs.aws.amazon.com/AmazonS3/latest/dev/DeletingObjectVersions.html